nlgeval/pycocoevalcap/meteor/meteor.py

```diff
@@ -101,6 +101,7 @@ def _stat(self, hypothesis_str, reference_list):
         # SCORE ||| reference 1 words ||| reference n words ||| hypothesis words
         hypothesis_str = hypothesis_str.replace('|||', '').replace('  ', ' ')
```

Comment:
You can probably now remove `.replace('  ', ' ')` from this line.

Reply:
fixed in e35f286. i missed the comment earlier :)